gwt-datetimeformat/src/test/java/org/gwtproject/i18n/jvm/DateTimeFormat_en_Test.java

```diff
@@ -0,0 +1,574 @@
+  //        "MM/dd/yyyy HH:mm:ss z").format(date, usPacific));
+  //    date.setTime(Date.UTC(2006 - 1900, 10 - 1, 29, 9, 0, 0));
+  //    assertEquals("10/29/2006 01:00:00 PST", DateTimeFormat.getFormat(
+  //        "MM/dd/yyyy HH:mm:ss z").format(date, usPacific));
+  //  }
+
+  public void test_EEEEMMMddHHmmsszzzyy() {
+    DateTimeFormat dtf = DateTimeFormat.getFormat("EEE MMM dd HH:mm:ss zzz yyyy");
+    Date date = dtf.parse("Mon Aug 30 00:45:00 GMT 2010");
+    assertEquals(45, (date.getHours() * 60 + date.getMinutes() + date.getTimezoneOffset()) % 1440);
+    date = dtf.parse("Mon Aug 30 01:45:00 GMT 2010");
+    assertEquals(105, (date.getHours() * 60 + date.getMinutes() + date.getTimezoneOffset()) % 1440);
+    date = dtf.parse("Mon Aug 30 06:45:00 GMT 2010");
+    assertEquals(405, (date.getHours() * 60 + date.getMinutes() + date.getTimezoneOffset()) % 1440);
+    date = dtf.parse("Mon Aug 30 07:45:00 GMT 2010");
+    assertEquals(465, (date.getHours() * 60 + date.getMinutes() + date.getTimezoneOffset()) % 1440);
+  }
+
+  public void test_EEEEMMMddyy() {
+    Date date = new Date(2006 - 1900, 6, 27, 13, 10, 10);
+    assertEquals(
+        "Thursday,July 27, 2006", DateTimeFormat.getFormat("EEEE,MMMM dd, yyyy").format(date));
+  }
+
+  public void test_EEEMMMddyy() {
+    Date date = new Date(2006 - 1900, 6, 27, 13, 10, 10);
+    assertEquals("Thu, Jul 27, 06", DateTimeFormat.getFormat("EEE, MMM d, yy").format(date));
+  }
+
+  public void test_HHmmss() {
+    Date date = new Date(2006 - 1900, 6, 27, 13, 10, 10);
+    assertEquals("13:10:10", DateTimeFormat.getFormat("HH:mm:ss").format(date));
+  }
+
+  public void test_hhmmssa() {
+    Date date = new Date(2006 - 1900, 6, 27, 13, 10, 10);
+    assertEquals("1:10:10 PM", DateTimeFormat.getFormat("h:mm:ss a").format(date));
+  }
+
+  public void test_k() {
+    DateTimeFormat dtf = DateTimeFormat.getFormat("yyyy-MM-dd kk:mm:ss");
+    Date date;
+    TimeZone utc = TimeZone.createTimeZone(0);
+
+    date = new Date(Date.UTC(2013 - 1900, 1 - 1, 1, 0, 34, 56));
+    assertEquals("2013-01-01 24:34:56", dtf.format(date, utc));
+    date = new Date(Date.UTC(2013 - 1900, 1 - 1, 1, 1, 34, 56));
+    assertEquals("2013-01-01 01:34:56", dtf.format(date, utc));
+  }
+
+  public void test_LL() {
+    Date date = new Date(2006 - 1900, 6, 27, 13, 10, 10);
+    assertEquals("07", DateTimeFormat.getFormat("LL").format(date));
+  }
+
+  public void test_LLL() {
+    Date date = new Date(2006 - 1900, 6, 27, 13, 10, 10);
+    assertEquals("Jul", DateTimeFormat.getFormat("LLL").format(date));
+  }
+
+  public void test_LLLL() {
+    Date date = new Date(2006 - 1900, 6, 27, 13, 10, 10);
+    assertEquals("July", DateTimeFormat.getFormat("LLLL").format(date));
+  }
+
+  public void test_LLLLL() {
+    Date date = new Date(2006 - 1900, 6, 27, 13, 10, 10);
+    assertEquals("J", DateTimeFormat.getFormat("LLLLL").format(date));
+  }
+
+  //  public void test_predefinedFormat() {
```

Comment:
?

(last time i'll write that this review, i think we need to put a note, possibly a filed issue, etc, but we don't want to have commented out code that can rot, be forgotten to be fixed, etc)